Servers/UI/OJS.Servers.Ui/ClientApp/src/components/home-contests/contest-card/ContestCard.module.scss

```diff
@@ -49,3 +55,32 @@
   font-weight: bold;
   color: $text-light-gray-color;
 }
+
+.contestCardTitle {
+  white-space: nowrap;
+  overflow: hidden;
+  display: block;
+  text-overflow: ellipsis;
+  width: 100%;
+  max-width: 225px;
+}
+
+.tooltip {
+  position: absolute;
+  display: none;
+  border-bottom: 1px dotted black;
```

Comment:
We almost have no elements using black. We have dark-gray color for this in the colors scss module